homeassistant/components/remote/__init__.py

```diff
@@ -59,6 +61,7 @@
     vol.Optional(
         ATTR_NUM_REPEATS, default=DEFAULT_NUM_REPEATS): cv.positive_int,
     vol.Optional(ATTR_DELAY_SECS): vol.Coerce(float),
+    vol.Optional(ATTR_HOLD_SECS, default=DEFAULT_HOLD_SECS): vol.Coerce(float),
```

Comment:
Changing the entity model requires approval in an issue in our architecture repo.

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model

Reply:
@MartinHjelmare ,

Looking for some advice is this is indeed required or nor. The idea here is that when the service remote.send_command is called that one can also include a parameter for how long to "hold" the button before releasing it.
Currently this would work with Harmony remotes but I cannot say for any others.

To be able to add this parameter, is this thus the right approach? Or is it possible to only have this parameter to be valid when the entity is for the Harmony remote?

Here is an example of what it thus looks like (in this case it would be to hold the "button" for 3 seconds):
```
left_blind_up:
  alias: "Left blind Up"
  sequence:
    - service: remote.send_command
      entity_id: remote.family_room
      data:
        command:
           - Up2
        device: 37297816
        hold_secs: 3
```

Your input is very much appreciated. :-)